protocol/native/native.go

```diff
@@ -36,7 +38,28 @@ func ExecParser(toNoder ToNoder, path string, args ...string) (*Parser, error) {
 }
 
 func (p *Parser) ParseUAST(req *protocol.ParseUASTRequest) *protocol.ParseUASTResponse {
-	nativeResp, err := p.Client.ParseNativeAST(&ParseASTRequest{Content: req.Content})
+	// If the request encoding is Base64 try to decode it as UTF8. Leave it
+	// as Base64 if it isn't possible.
```

Comment:
It was meant to cover cases of the client encoding stuff to Base64 that could really be encoded without problems in UTF8 (think most language specific pre unicode encodings, ISO-8859-X, etc) but @alcortesm got a good point about possible changing file line positions which could bit us if we convert to UTF8 and the original encoding was different (and used code points for positions instead of graphemes for example), so I'm disabling this on my next commit.